packages/react/src/components/Table/TableBody/TableBodyLoadMoreRow/TableBodyLoadMoreRow.test.jsx

```diff
@@ -2,23 +2,55 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
+import { settings } from '../../../../constants/Settings';
+
 import TableBodyLoadMoreRow from './TableBodyLoadMoreRow';
 
-const tableRowProps = {
+const { prefix } = settings;
+
+const getTableRowPropsNotLoading = () => ({
```

Comment:
again, just a note, since you're already using a function, you could just make it take a parameter to override any of the defaults. ie:

```suggestion
const getTableRowPropsNotLoading = (overrides) => ({
```

and then add the overrides with a spread operator at the bottom of the object:

```
...overides,
});
```